array-manipulation-2/src/drop.js

```diff
@@ -0,0 +1,10 @@
+/* exported drop */
+function drop(array, count) {
+  var newArray = [];
+  for (var i = 0; i < array.length; i++) {
+    if (i >= count) {
+      newArray.push(array[i]);
+    }
```

Comment:
You can simplify this by setting `var i = count` as the initialization expression of your loop.